mne_features/feature_extraction.py

```diff
@@ -360,19 +365,39 @@ class ``FeatureExtractor`` can be used as a step in a Pipeline (see
     :func:`extract_features`
     """
 
-    def __init__(self, sfreq=256., selected_funcs=None, params=None, n_jobs=1,
-                 memory=None):
+    def __init__(self, sfreq=256., ch_names=None, selected_funcs=None,
```

Comment:
this is an API breakage. To minimize the risk of breakage maybe you can put the param ch_names just before n_jobs?

Reply:
my bad, I will update it